### ISSUE TITLE
daily note and quick note on https://silverbullet.md

### ISSUE THREAD

**Issue by davidbeijinho** [silverbulletmd/silverbullet#535]
when running the commands: 
- `Quick Note`
- `Open Daily Note` 

html is returned and not not a new page is created, i tested on docker version and it works ok

![image](https://github.com/silverbulletmd/silverbullet/assets/2984693/940b3e04-f4b4-4a99-81ab-35f6679292c9)

**Comment by zefhemel** [comment]
Yeah, silverbullet.md is not supposed to be a fully working version (it's mostly a demo), but still this is not very nice. I'll see if I can fix this.

**Comment by zefhemel** [comment]
Now available on https://play.silverbullet.md
